packages/dd-trace/src/ritm.js

```diff
@@ -117,8 +117,16 @@ function Hook (modules, options, onrequire) {
         // abort if _resolveLookupPaths return null
         return exports
       }
-      const res = Module._findPath(name, [basedir, ...paths])
-      if (res !== filename) {
+
+      let res
```

Comment:
Unsure about this change - basically, certain versions of `next` don't copy over the "main" entry point file specified in its `package.json` to the standalone directory, which functionally doesn't really matter because it's not used explicitly in the auto-generated server file, and the server file requires a module-internal library. So, in that context, this change is OK, but maybe not in the general context? But, in what other context would the file specified as the main entry point in its `package.json` not exist?